media/css/mozorg/moss/common.scss

```diff
@@ -364,6 +364,7 @@ main {
 // index page
 #moss-home #header-image {
     background: #d1d2d3;
+    display: none;
```

Comment:
Last ask - please remove the other CSS in this selector. Since the element is being hidden with `display: none;`, we don't need `background` set or the media query below. 

Thanks!